OracleLinuxDevelopers/oraclelinux8/redis/5/Dockerfile

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM ghcr.io/oracle/oraclelinux:8
+
+RUN dnf install -y redis && \
+    rm -rf /var/cache/dnf
+
+RUN /bin/bash -o pipefail -c "sed 's/protected-mode yes/protected-mode no/g' /etc/redis.conf | \
+    sed 's/^bind/#bind/g' - | sed 's/^logfile .*/logfile \"\"/g' - > /etc/docker.conf"
```

```suggestion
SHELL ["/bin/bash", "-c", "-o", "pipefail"]

RUN sed -i.bak \
    -e 's/protected-mode yes/protected-mode no/g' \
    -e 's/^bind/# bind/g' \
    -e 's/^logfile .*/logfile \"\"/g' \
    /etc/redis.conf
```

"Docker" is a trademark of Docker, Inc and thus can only be used when referring specifically to Docker, Inc. or one of their products, so the suggestion is to make the changes in-place to `/etc/redis.conf` instead while creating a backup  of the original config too. 

I moved the bash stuff to a `SHELL` directive to make things easier to read and should go before all the `RUN` directives so that it applies to all of them.

btw, I did raise an eyebrow at the combination of disabling protected mode and binding to all interfaces, but then saw the official image does it too (by patching the source code, no less) so I guess it's fine.

---

Note that we don't need the `pipefail` stuff anymore since the pipe is gone ;)

---

Ha, I totally missed that. 😂 That's what I get for reviewing PRs before 8am in the morning.

---

I didn't actually know the SHELL directive existed.. will keep in mind for future.

Also didn't realise there was a dnf module for Redis, so will change that too.

---

Have made the sed changes and added a 6/Dockerfile for both versions. Nice